docs/use-cases/deploy-code.md

```diff
@@ -1,8 +1,73 @@
+
+Once created and uploaded, you can update and manage module versions by using the [modular registry](https://app.viam.com/registry) to ensure your modules are accessible and kept up to date.
+
+You can also add an existing module from the [Viam registry](https://app.viam.com/registry), or create your own.
+
+With modules, it is easy to deploy code to a robot or a fleet of robots with robust control over code versioning.
+
+<table>
+  <tr>
+    <th>{{<imgproc src="/ml/collect.svg" class="fill alignleft" style="max-width: 150px" alt="ml collect icon">}}
+      <b>1. Search modules</b>
+      <br><br>
+      <p>First, <a href="/fleet/machines/#add-a-new-robot">create a robot</a> if you haven't yet.</p>
+      <p><a href="/registry/configure/">Search for modules</a> that fits your robot's requirements in the Viam Registry and <a href="https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry">add it</a> from your robot's page.</p>
+    </th>
+  </tr>
+  <tr>
+    <th>{{<imgproc src="/ml/configure.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="ml configure icon">}}
+      <b>2. Create your own module</b>
+      <br><br>
+      <p>If you don't find a module that meets your needs, you can <a href="https://docs.viam.com/registry/create/">create your own module</a> to add support for new hardware, or to extend an existing software service.</p>
+    </th>
+  </tr>
+  <tr>
+    <th>{{<imgproc src="/ml/deploy.svg" class="fill alignleft" style="max-width: 150px" declaredimensions=true alt="ml deploy icon">}}
+      <b>3. Deploy your module</b>
+      <br><br>
+      <p>Once you create your module, you can <a href="/registry/upload/">upload</a> it to the Viam registry for deployment. You have the option to customize several aspects of your deployment experience: </p></br>
+      <ul>
+        <li>
+          Deployment Visibility: You can tailor your module's visibility by <a href="https://docs.viam.com/registry/upload/#upload-a-custom-module">uploading it to the registry</a> as either a public or private module.
+              <ul>
+                <li>A public module is available to all Viam users.</li>
+                <li>A private module is visible only to members of your <a href="/fleet/organizations/">organization</a>.</li>
+              </ul>
+         </li>
+      </ul>
+      <ul>
+        <li>
+        Offline Deployment: If your robot was offline when you configured the module for use, it will deploy once your machine is online.
+        </li>
+        </br>
+        <li>
+        Local Module Deployment: You can easily deploy to a robot or fleet as a <a href="/registry/configure/#local-modules">local module</a> to develop your module locally before uploading it to the Viam registry.
+        </li>
+      </ul>
+  </tr>
+  <tr>
```

Comment:
Nit: some tweaks to layout and language:

```suggestion
      <p>Once you have created your new module, you can deploy it to your machine in one of two ways:</p><br><br>
      <ul><li>You can <a href="/registry/upload/">upload your module</a> to the Viam registry using the Viam CLI. Modules available from the Viam registry can be deployed directly to a machine or fleet of machines from the Viam app. When you upload your module to the registry, you can choose to make it:</li>
        <ul><li><b>Public:</b> your module is available to all Viam users.</li>
        <li><b>Private:</b> your module is only visible to members of your <a href="/fleet/organizations/">organization</a>. This is the default state of new modules.</li></ul></li>
      <li>You can deploy your module directly to your machine as a <a href="/registry/configure/#local-modules">local module</a>. Local modules are not uploaded to the Viam registry, and must be manually added to your machine.</li>
      </ul>
      <p> If your machine is offline when you deploy a module, it will deploy once your machine comes back online.</p>
    </th>
  </tr>
  <tr>
```

Here I try to:
- Avoid UL breaks due to image align wrapping 😅  - not a perfect solution, but we'll be revisiting use case page layout consistency in the near future, so OK for now IMO.
- Stress: two methods of deploy, with pub/priv being second order considerations, scoped only to one method. Previously, Offline deploy seemed like an alternative to Local, etc.
- Briefly mention the CLI, but not linked (we want users to click through to the `upload` steps, which fully document everything the user needs to know about CLI but scoped to module stuff only. Bare CLI link might confuse when the user lands on "how to upload your data CLI steps here" for example.